account/manager_test.go

```diff
@@ -531,8 +531,8 @@ func TestResumeAccountAfterRestart(t *testing.T) {
 	// persisted intent to create the account.
 	go func() {
 		_, _ = h.manager.InitAccount(
-			context.Background(), value, expiry, bestHeight,
-			DefaultFundingConfTarget,
+			context.Background(), value, chainfee.FeePerKwFloor, expiry,
```

Comment:
nit: should probably be moved to the previous commit.

Reply:
done